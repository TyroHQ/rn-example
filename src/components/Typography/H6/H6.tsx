import React from "react";
import { Text } from "react-native";
import styled from "styled-components/native";

const H6Container = styled.View`
  background-color: lightgrey;
  padding: 10px;
`;

export class H6 extends React.Component<{}> {
  static defaultProps: H6["props"];
  render() {
    return <Text>H6</Text>;
  }
}
