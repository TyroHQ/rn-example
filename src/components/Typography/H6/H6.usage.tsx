import React from "react";
import { boolean, text, number } from "@storybook/addon-knobs";
import { action } from "@storybook/addon-actions";
import { H6 } from "./";

export const basic = () => <H6 />;
