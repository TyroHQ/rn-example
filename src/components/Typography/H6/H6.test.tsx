import { shallow } from "enzyme";
import * as usage from "./H6.usage";

describe("<H6 />", () => {
  for (const example in usage) {
    it(`renders well in ${example} mode`, () => {
      const x = (usage as any)[example]();
      const wrapper = shallow(x);
      expect(wrapper).toMatchSnapshot();
    });
  }
});
