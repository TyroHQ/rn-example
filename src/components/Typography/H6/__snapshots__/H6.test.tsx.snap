// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<H6 /> renders well in basic mode 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
>
  H6
</Text>
`;
